docs/source/commands/peerlifecycle.md

```diff
@@ -116,7 +116,7 @@ Global Flags:
 Install a chaincode on a peer.
 
 Usage:
-  peer lifecycle chaincode install [flags]
+  peer lifecycle chaincode install [packageFiles] [flags]
```

Comment:
```suggestion
  peer lifecycle chaincode install [packageFile] [flags]
```

Reply:
I have corrected my changes and update with a new commit.

thanks again for your help